Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -3728,6 +3730,23 @@ public PlainTextString ErrorMessage
       set { _controlServiceArguments = StringUtility.EmptyToNull(value); }
     }
 
+    /// <summary>
+    /// Returns the <see cref="BocColumnDefinition"/>s displayed in the <see cref="BocList"/> that also support validation.
+    /// </summary>
+    /// <remarks>
+    /// The column definitions change when one of the following properties are changed:
+    /// <list type="bullet">
+    ///   <item><see cref="FixedColumns"/></item>
+    ///   <item><see cref="SelectedView"/></item>
+    ///   <item><see cref="SelectedViewIndex"/></item>
+    /// </list>
+    /// </remarks>
+    public IEnumerable<IBocColumnDefinitionWithValidationSupport> GetColumnsWithValidationSupport ()
+    {
+      var columns = EnsureColumnsGot();
+      return columns.OfType<IBocColumnDefinitionWithValidationSupport>();
+    }
```

Comment:
@a-ctor just did the `GetColumnDefinitions` API. Maybe we don't need the interface at all and rather do the filter at the callsite because the filter is primitive and it removes indirections/question marks when seeing multiple specialized APIs.

If we keep the method, move it to the other GetColumnDefinitions APIs please.

Reply:
And looking at the implem,entations / task, I think we don't actually need this api in this task? -> it would be a separate task because there's no dependency to it? So, we should move it into the PR where it's needed, then we can also check if it's actually required or just fancy code.